### PR TITLE
Timber1900.WebDL version v10

### PR DESCRIPTION
**Pull request by Timber1900**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/19644)

### CODE REVIEW

**Comment by ryfu-msft** [comment]
Hey @Timber1900,

Sorry for the delays on this pull request. I had issues running this application on an x86 host, but was able to successfully run it on an x64 host. Could you try changing the architecture to x64? Thanks!

**Comment by Timber1900** [comment]
> Hey @Timber1900,
> 
> Sorry for the delays on this pull request. I had issues running this application on an x86 host, but was able to successfully run it on an x64 host. Could you try changing the architecture to x64? Thanks!

The app does target x64, I chose the wrong option in wingetcreate, apologies

**Comment by Timber1900** [comment]
The installer is installing by default in Program Files(x86), i'll update it and see if it works

File: manifests/t/Timber1900/WebDL/v10/Timber1900.WebDL.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: Timber1900.WebDL
+PackageVersion: v10
+PackageLocale: en-US
+Publisher: Timber1900
+PackageName: WebDL
+License: MIT License
+ShortDescription: WebDL Setup
```
Comment (ItzLevvie) [inline manifests/t/Timber1900/WebDL/v10/Timber1900.WebDL.locale.en-US.yaml:9]:
```
ShortDescription: WebDL Setup
```
Write a better description.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by OfficialEsco — APPROVED** [review]
Would recommend to update your wingetcreate :)

**Pull request merged**